### DISCUSSION TITLE
Feature Request: Button to edit current note on the git repo

### DISCUSSION THREAD

**Discussion by David7ce** [imfing/hextra#48] [Ideas]
It would be great add an option to edit GitHub note to act as a wiki.

A button that redirects to edit current page forking the repo
<a href="https://github.com/imfing/hextra/edit/main/README.md"> Edit page </a>. And then the editor just make a "Pull Request" to merge changes with default main branch.

And it should be located at the top right to be more intuitive and easier to find, like Wikipedia and other wikis do.

Btw great theme!

**Comment by imfing** [comment]
Thanks. 

Currently there's an option to configure the edit page link on the right bottom:

https://imfing.github.io/hextra/docs/guide/configuration/#page-edit-link

**Reply by imfing** [reply]
and I think we can support custom shortcode on the top right, so you can put a button there which takes directly to the edit page.
